test/piezo.js

```diff
@@ -181,222 +192,55 @@ exports["Piezo"] = {
+      test.done();
+    });
   },
 
-  play: function(test) {
-    test.expect(3);
+  stop: function(test) {
+    test.expect(2);
 
-    var returned = this.piezo.play({
-      song: [
-        [] // No tone
+    var returned = this.piezo.stop();
 
-      ],
-      tempo: 150
-    });
     test.ok(this.spy.calledWith(3, 0));
     test.equal(returned, this.piezo);
 
-
-    this.piezo.play({
-      song: [
-        [] // No tone
-      ],
-      tempo: 150
-    });
-    test.ok(this.spy.calledWith(3, 0));
-
     test.done();
-  },
-
-  playTune: function(test) {
-    var tempo = 10000; // Make it really fast
-    test.expect(6);
-    var freqSpy = sinon.spy(this.piezo, "frequency");
-    this.piezo.play({
-      song: [
-        ["c", 1],
-        ["d", 2],
-        [null, 1],
-        672,
-        "e4",
-        null
-      ],
-      tempo: tempo // Make it real fast
-    }, function() {
-      // frequency should get called 4x; not for the null notes
-      test.ok(freqSpy.callCount === 4);
-      test.ok(freqSpy.neverCalledWith(null));
-      // First call should have been with frequency for "c4"
-      test.ok(freqSpy.args[0][0] === Piezo.Notes["c4"]);
-      // Default duration === tempo if not provided
-      test.ok(freqSpy.calledWith(Piezo.Notes["e4"], 60000 / tempo));
-      // Duration should change if different beat value given
-      test.ok(freqSpy.calledWith(Piezo.Notes["d4"], (60000 / tempo) * 2));
-      // OK to pass frequency directly...
-      test.ok(freqSpy.calledWith(672, 60000 / tempo));
-      test.done();
-    });
-  },
-
-  playTuneWithStringSongAndBeat: function(test) {
-    var tempo = 10000; // Make it really fast
-    test.expect(6);
-    var freqSpy = sinon.spy(this.piezo, "frequency");
-    var beats = 0.125;
-    this.piezo.play({
-      song: "c d d - 672 e4 -",
-      beats: beats,
-      tempo: tempo // Make it real fast
-    }, function() {
-      // frequency should get called 4x; not for the null notes
-      test.ok(freqSpy.callCount === 4);
-      test.ok(freqSpy.neverCalledWith(null));
-      // First call should have been with frequency for "c4"
-      test.ok(freqSpy.args[0][0] === Piezo.Notes["c4"]);
-      // Default duration === tempo if not provided
-      test.ok(freqSpy.calledWith(Piezo.Notes["e4"], 60000 * beats / tempo));
-      // Duration should change if different beat value given
-      test.ok(freqSpy.calledWith(Piezo.Notes["d4"], (60000 * beats / tempo) * 2));
-      // OK to pass frequency directly...
-      test.ok(freqSpy.calledWith(672, 60000 * beats / tempo));
-      test.done();
-    });
-  },
-
-  playCanDealWithWonkyValues: function(test) {
-    var tempo = 10000,
-      tune = {
-        song: [
-          ["c4"],
-          ["drunk"],
-          ["d4", 0]
-        ],
-        tempo: tempo
-      };
-    test.expect(1);
-
-    this.piezo.play(tune, function() {
-      test.ok(1); // We made it this far, no choking on bad values
-      test.done();
-    }.bind(this));
-
-  },
```

Comment:
There are a lot more tests to come, i just have the bear minimum to get the octvian work in.